### PR TITLE
795 bugfixes

### PR DESCRIPTION
**Pull request by ajrbyers**
Trello card: https://trello.com/c/W7ju84Xk/494-795-132-bugs#
Issue: #795

### CODE REVIEW

**Comment by ajrbyers** [comment]
[#795: 1.3.2 Bugs](https://trello.com/c/W7ju84Xk/494-795-132-bugs)

File: src/copyediting/forms.py [hunk 1]
```diff
@@ -25,3 +25,9 @@ class AuthorCopyeditForm(forms.ModelForm):
     class Meta:
         model = models.AuthorReview
         fields = ('decision', 'author_note')
+
+    def __init__(self, *args, **kwargs):
+        # first call parent's constructor
```
Comment (ajrbyers) [inline src/copyediting/forms.py]: Comments have been dropped in future commit fyi.

File: src/core/views.py [hunk 2]
```diff
@@ -866,7 +866,7 @@ def add_user(request):
     form = forms.EditAccountForm()
     registration_form = forms.AdminUserForm(active='add', request=request)
     return_url = request.GET.get('return', None)
-    role = request.GET.get('role', None)
```
Comment (mauromsl) [inline src/core/views.py]: We are only assigning the role if another role is not defined in `request.GET["role"]`. This seems to contradict the commit title. Is this what we intended?

Comment (ajrbyers) [reply src/core/views.py]: Indeed! Commit incoming.

**Pull request merged**